release-notes/2023-10/README.md

```diff
@@ -0,0 +1,252 @@
+<!-- This file should be renamed to README.md and placed in the directory for the release. -->
+
+# SystemLink Enterprise release 2023-10 Release Notes
+
+The 2023-10 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## Upgrading from the release 2023-09 to the release 2023-10
+
+<!-- Optional section to include comments and instructions needed to successfully upgrade from the previous release to the current release. If the only changes needed are already captured in Helm Chart Breaking Changes, this section is not needed. -->
+
+## New Features and Behavior changes
+
+- webserver 0.13.4
+    - Upgraded Redis dependency from 7.0 to 7.2. This is a breaking change. It is necessary to upgrade the entire cluster in parallel, which is not something Kubernetes will do automatically.
+    - Option #1: Set webserver.redis-cluster.redis.update-strategy.type = OnDelete
+        - Run the upgrade deployment
+        - Run kubectl -n <`namespace`> delete pods <`release`>-webserver-redis-0 <`release`>-webserver-redis-1 <`release`>-webserver-redis-2 <`release`>-webserver-redis-3 <`release`>-webserver-redis-4 <`release`>-webserver-redis-5
+        - The pods of the stateful set will be deleted and should be automatically recreated in parallel.
+        - Remove the overide of the redis update-strategy from the configuration. You can re-deploy to apply this change but it is not required.
+    - Option #2: Prior to upgrade, run: kubectl -n <`namespace`> delete statefulset <`release`>-webserver-redis
+        - This will delete the redis cluster, preventing UI access to the application.
+        - Now run the upgrade deployment. The redis cluster will be recreated and deployed in parallel.
+    - Once upgraded, Redis storage will be incompatibile with older versions of the software. If it is necessary to downgrade to an older version, you must perform a hard reset on the redis cluster:
+        - [Perform-a-hard-reset-on-the-redis-cluster.md](https://github.com/ni/install-systemlink-enterprise/tree/2023-10/release-notes/2023-10/Perform-a-hard-reset-on-the-redis-cluster.md)
+- dataframeservice 0.14.14
+    - The Dremio data set refresh job interval was increased from 2 minutes to 1 hour. This reduces overall load on Dremio.
+    - Customers are not required to uptake this change, but doing so will be beneficial as the net result in load reduction.
+    - The following steps cover what the customer needs to do:
+        - [How-to-reset-Dremio.md](https://github.com/ni/install-systemlink-enterprise/tree/2023-10/release-notes/2023-10/How-to-reset-Dremio.md)
+- dataframeservice 0.14.39
```

Comment:
Something isn't right here. We can't both be shipping DFS 0.14.4 and 0.14.39.

Reply:
The version we ship is 0.14.49, so I put that instead; the earlier versions are internal versions when changes were made but which are older versions that 0.14.49 that we ship